main/runner.py

```diff
@@ -0,0 +1,89 @@
+# source: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/appengine/standard/localtesting/runner.py
+# license: https://www.apache.org/licenses/LICENSE-2.0
+
+# [START runner]
```

Comment:
```suggestion
```

Reply:
Any idea why Google had these comments in their code?

Reply:
Not really.. maybe they just use a lot of comments.. do you think it could be used somewhere?

Reply:
Can't see where. I suspect it is a hint to whatever they use to embed the code in their web page. (So we should be able to safely remove.)